jaeger-crossdock/docker-compose.yml

```diff
@@ -35,40 +35,73 @@ services:
 
     go:
         image: jaegertracing/xdock-go
+        links:
+            - "jaeger:jaeger-agent"
         ports:
             - "8080-8082"
 
     python:
         image: jaegertracing/xdock-py
+        links:
+            - "jaeger:jaeger-agent"
         ports:
             - "8080-8082"
 
     nodejs:
         image: jaegertracing/xdock-node
+        links:
+            - "jaeger:jaeger-agent"
         ports:
             - "8080-8082"
 
     java-udp:
         depends_on:
             - test_driver
```

Comment:
I think this needs to depend on `jaeger`, if you look at PRs for the other clients

Reply:
Yes, you are right. I think earlier it was like `jave-udp -> test_driver -> (jaeger components)`.